src/cloudant-util.js

```diff
@@ -0,0 +1,32 @@
+const {CLOUDANT_API_KEY, CLOUDANT_URL} = require("./config.json");
```

Comment:
The credentials are hidden through .gitignore right? Just want to double check this before we merge it to master.

Reply:
If it is already in .gitignore, then you can go ahead and merge. Thank you for helping me with Cloudant integration again.